### DISCUSSION TITLE
How to disable current code context on the top?

### DISCUSSION THREAD

**Discussion by GreenSleeper** [LazyVim/LazyVim#2469] [Q&A]
Lazyvim shows the current code context I'm in like it would show the function name on the top when I'm in a really long function.
How can I disable this feature?

**Comment by dpetka2001** [comment]
```lua
return {
  { "nvim-treesitter/nvim-treesitter-context", enabled = false }
}
```
in a file inside the `~/.config/nvim/lua/plugins/` directory.

**Reply by GreenSleeper** [reply]
solved, thanks.
